src/components/NoteList/index.tsx

```
@@ -1,28 +1,29 @@
-import { cn } from '@/lib/utils';
-import { NoteItem } from './NoteItem';
+import { cn } from "@/lib/utils";
+import { NoteItem } from "./NoteItem";
+import { useNoteStore } from "@/modules/notes/note.state";
 
 interface NoteListProps {
   layer?: number;
   parentId?: number;
 }
 
 export function NoteList({ layer = 0, parentId }: NoteListProps) {
```

ESLint

[error] 'parentId' is defined but never used.

(@typescript-eslint/no-unused-vars)